apps/tools/scxmldump/main.cpp

```diff
@@ -124,6 +135,8 @@ class EventDump : public Seiscomp::Client::Application {
 			                        "Export focal mechanisms.");
 			commandline().addOption("Dump", "ignore-arrivals,a",
 			                        "Do not export origin arrivals.");
+			commandline().addOption("Dump", "ignore-unused",
```

Comment:
The name of the option is misleading and the description is wrong since the weight is tested and not timeUsed, horizontalSlownessUsed, backazimuthUsed [1].

[1] https://docs.gempa.de/seiscomp/current/base/api-python.html#arrival